tests/foreman/ui/test_classparameters.py

```diff
@@ -1165,6 +1165,43 @@ def test_positive_create_matcher_attribute_priority(self):
             self.assertEqual(output_scp, override_value)
 
     @run_only_on('sat')
+    @tier1
+    def test_positive_update_with_long_priority_list(self):
+        """Smart class parameter priority order list can contain more than 255
+        character inside of it
+
+        :id: f5e7847a-1f4b-455d-aa73-6b02774b6168
+
+        :steps:
+            1.  Check the Override checkbox.
+            2.  Set some default Value.
+            3.  Set long priority order list
+
+        :expectedresults: Smart class parameter is updated successfully and has
+            proper priority list
+
+        :BZ: 1458817
+
+        :CaseImportance: Medium
+        """
+        sc_param = self.sc_params_list.pop()
+        order_value = '\n'.join(
+            [gen_string('alpha').lower() for _ in range(40)])
```

Comment:
can we make an assert that value is more than 255 chars long, just to be to be sure, as gen_string without length,
also  order_value is of length  439 chars, can we use the same approximate length as the bug was verified with eg: 604 chars